src/test/kotlin/com/example/mykku/config/TestImageUploadConfig.kt

```
@@ -65,6 +65,9 @@ class TestImageUploadConfig {
                     imageUrls = imageUrls
                 )
             }
+
+            override fun delete(url: String) {
+            }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)